AttributeCategoryForCAP/examples/example.g

```diff
@@ -119,7 +119,7 @@ structure_record.DualOnObjects :=
 ## In this particular example, this function would be not necessary since it can be 
 ## derived from the rigidity of the category
 structure_record.InternalHomOnObjects :=
-  function( object1, object2, underlying_internal_hom )
+  function( cat, object1, object2, underlying_internal_hom )
```

Comment:
There is still a `cat` here, is this deliberate?

Reply:
Argh... no, it was probably too close to the `AddIsEqual...` below, so I overlooked it.
Thank you for spotting it.